mmv1/products/securityposture/PostureDeployment.yaml

```diff
@@ -0,0 +1,159 @@
+update_verb: :PATCH
+update_mask: true
+
+
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'securityposture_posture_deployment_basic'
+    primary_resource_id: 'postureDeployment'
+    test_env_vars:
+      org_id: :ORG_ID
+      project_number: :PROJECT_NUMBER
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: parent
+    description: |
+      The parent of the resource, an organization. Format should be `organizations/{organization_id}`.
+    immutable: true
+    url_param_only: true
+    required: true
+  - !ruby/object:Api::Type::String
+    name: location
+    description: |
+      The location of the resource, eg. global`.
+    immutable: true
+    url_param_only: true
+    required: true
+  - !ruby/object:Api::Type::String
+    name: postureDeploymentId
+    description: |
+      ID of the posture deployment.
+    immutable: true
+    required: true
+    url_param_only: true
+
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description: |
+      Name of the posture deployment instance.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'targetResource'
+    required: true
+    immutable: true
+    description: |
+      The resource on which the posture should be deployed. This can be in one of the following formats:
+      projects/<project_number>
+      folders/<folder_number>
+      organizations/<organization_id>
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    description: |
+      State of the posture deployment.
+    output: true
+    values:
+      - :STATE_UNSPECIFIED
+      - :CREATING
+      - :DELETING
+      - :UPDATING
+      - :ACTIVE
+      - :CREATE_FAILED
+      - :UPDATE_FAILED
+      - :DELETE_FAILED
+  - !ruby/object:Api::Type::String
+    name: 'postureId'
+    required: true
+    description: |
+      Relative name of the posture which needs to be deployed. It should be in the format:
+        organizations/<ORG_ID>/locations/<LOCATION>/postures/<postureID>
+  - !ruby/object:Api::Type::String
+    name: 'postureRevisionId'
+    required: true
+    description: |
+      Revision_id the posture which needs to be deployed.
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      Time the posture deployment was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      Time the posture deployment was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      Description of the posture deployment.
+  - !ruby/object:Api::Type::String
+    name: 'etag'
+    description: 'For Resource freshness validation (https://google.aip.dev/154)'
+    output: true
+  - !ruby/object:Api::Type::KeyValueAnnotations
+    name: 'annotations'
```

Comment:
qq: why do we need to send empty value for this field?

Reply:
Had taken reference from other resources. But accepting empty values is not important for this. Have removed it.